vars/createSctRunner.groovy

```diff
@@ -21,20 +21,29 @@ def call(Map params, Integer test_duration, String region) {
     } else {
         availability_zone = params.availability_zone
     }
+
+
     if ( availability_zone ) {
         availability_zone_arg = "--availability-zone " + availability_zone
     }
+
+    if ( params.backend.equals("azure") ) {
+        region_zone_arg = "--region " + params.azure_region_name
+    } else {
+        region_zone_arg = "--region " + region
+    }
+
     println(params)
     sh """
     #!/bin/bash
     set -xe
     env
 
-    if [[ "$cloud_provider" == "aws" || "$cloud_provider" == "gce" || "$cloud_provider" == "azure" ]]; then
+    if [ "$cloud_provider" = "aws" ] || [ "$cloud_provider" = "gce" ] || [ "$cloud_provider" = "azure" ]; then
```

Comment:
Why do you need these changes? On line 38 we know for sure that we run it in bash and it must work there.
Why?